src/products/views.py

```diff
@@ -41,4 +41,3 @@ def list_view(request):
     return render(request, template, context)
 
 
-
```

Comment:
this is not an actual change and should not be committed  in master. 
For consistency we all should avoid that kind of unnecessary changes (that are not code changes)   
You need to revert back this one line that was removed